src/mpi4py/MPI/msgbuffer.pxi

```diff
@@ -81,14 +81,41 @@ cdef inline bint is_IN_PLACE(object obj):
 
 #------------------------------------------------------------------------------
 
+cdef inline int is_bigendian() nogil:
+    cdef int i = 1
+    return (<char*>&i)[0] == 0
+
+cdef inline const char *getformat(const char format[]) except NULL:
+    cdef char byteorder = format[0]
+    if byteorder == c'@': # native
+        return format + 1
+    if byteorder == c'<': # little-endian
+        if is_bigendian(): raise ValueError(
+            f"format string {pystr(format)!r} "
+            f"with non-native byte order")
+        return format + 1
+    if byteorder == c'>': # big-endian
```

Comment:
```suggestion
    elif byteorder == c'>': # big-endian
```

Reply:
Same thing here, just a matter of style.